add a warning on media link attribute type (#555)
akeneo/pim-helpcenter: content/md/first-steps/manage-asset-families.md

The prefix and suffix are not mandatory.
:::

::: warning

> The PIM cannot ensure that what you currently see in a `media link` attribute is still what is externally stored. That's why we strongly recommend to use one url for one media. **As soon as your media that is stored externally comes to change, the best practice is to change its url in the PIM**. 

Reviewer: ```suggestion
The PIM cannot ensure that what you are currently seeing in a `media link` attribute is still what is externally stored. That's why we strongly recommend using one url per one media. **As soon as your media stored externally changes, the best practice is to change its url in the PIM**. 
```
